CMake/test_labels.cmake

```diff
@@ -0,0 +1,15 @@
+
+FUNCTION( ADD_TEST_LABELS TEST_NAME TEST_LABELS )
```

Comment:
Is this function needed?  It doesn't seem very useful for miniqmc - there's not a wide variety of tests to manage.

Reply:
I was thinking of at least 4 categories.
1. unit tests.
2. check_XXX tests
3. performance tests running miniqmc
4. we may need deterministic test for miniQMC as well.
So I think keep this label infrastructure can be useful. Making a difference between miniQMC and QMCPACK adds maintenance burden and let us keep this for now and delete if it causes problems.